systemd/cert-renewer@.service

```diff
@@ -22,7 +22,7 @@ ExecStart=/usr/bin/step ca renew --force ${CERT_LOCATION} ${KEY_LOCATION}
 
 ; Try to reload or restart the systemd service that relies on this cert-renewer
 ; If the relying service doesn't exist, forge ahead.
-ExecStartPost=/usr/bin/env bash -c "if ! systemctl --quiet is-enabled %i.service ; then exit 0; fi; systemctl try-reload-or-restart %i"
+ExecStartPost=/usr/bin/env sh -c "! systemctl --quiet is-enabled %i.service || systemctl reload-or-try-restart %i.service"
```

Comment:
this should be `systemctl try-reload-or-restart`not `systemctl reload-or-try-restart`

Reply:
and I think `is-enabled` should be `is-active`

Reply:
@tashian It used to be `reload-or-try-restart`, and was changed to `try-reload-or-restart` [in systemd 229][systemd]. Unfortunately, CentOS 7 uses systemd 219. The good news is that the old command is still accepted for backwards compatibility purposes.

[systemd]: https://github.com/systemd/systemd/blob/v229/NEWS#L115-L118

Reply:
Thanks @elyscape for elaborating on that. It's been tricky to balance future-proofing with broad compatibility on our systemd stuff. After some research in January, I found that v229 is a good minimum version for us to support, and v239+ is what a lot of distros are running. That said, I've added a comment (in a067b3a) to the file about using `reload-or-try-restart` in <229.

Reply:
Sounds good!